trade-finance-manager-ui/server/routes/banks/index.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const { validateUserTeam } = require('../../middleware');
+const { PDC_TEAM_IDS } = require('../../constants');
+const { getReportDownload } = require('../../controllers/utilisation-reports/report-download');
+
+const router = express.Router();
+
+router.get('/:bankId/utilisation-report-download/:_id', validateUserTeam(Object.values(PDC_TEAM_IDS)), (req, res) => getReportDownload(req, res));
```

Comment:
In contrast to Portal, I don't think we really care about the bank ID as we don't need to validate it against the user. I think the `banks/:bankId` part of the route could probably be removed, and this route moved to the `utilisation-reports` routes. Let me know if you disagree.

Reply:
Route now updated as described above.